src/obr/OpenFOAM/case.py

```diff
@@ -312,6 +333,12 @@ def decomposePar(self, args={}):
         fvSolutionArgs = args.get("fvSolution", {})
         if fvSolutionArgs:
             self.fvSolution.set(fvSolutionArgs)
+
+        if not find_time_folder(self.path / "processor0"):
+            logging.warning(
+                f"No time in processor folder found. This indicates an unusable"
+                f" decomposition."
+            )
```

Comment:
if it is unusable, should we continue with calculations ?

Reply:
I assume we should continue, since we don't really know if the user has a different way to provide the time folders later.